CMakeLists.txt

```diff
@@ -1,6 +1,15 @@
 cmake_minimum_required( VERSION 3.0 FATAL_ERROR )
 set( CMAKE_VERBOSE_MAKEFILE ON )
 
+# Configure the Android toolchain before the project start
+if( CINDER_TARGET )
```

Comment:
I think this should be a part of [configure.cmake](https://github.com/cinder/Cinder/blob/524c705c36adbf9644209d2210dc7dec26685f98/proj/cmake/configure.cmake), which sets up the `CINDER_TARGET` along with other important variables used on all platforms.

But perhaps the `cinderAndreoidToolchain.cmake` __must__ be included before the call to `project( cinder )`? If that is the case, I think that the `project()` call can be moved to after the `include( configure.cmake )`.